docker-compose.yml

```diff
@@ -3,4 +3,4 @@ services:
   tests:
     image: buildkite/plugin-tester
     volumes:
-      - ".:/plugin:ro"
+      - ".:/plugin"
```

Comment:
I had to make this `rw`, as we download scripts via `curl` into a plugin directory.

Reply:
Can we confirm this is going to be a writable dir in real life? I know in the [other plugin](https://github.com/joscha/sauce-connect-buildkite-plugin/blob/master/hooks/pre-command#L8) the upstream maintainer mentioned he was using `/tmp`

Reply:
changed it to `/tmp/codecov-bash` instead